client/lib/happychat/connection.js

```diff
@@ -105,9 +105,17 @@ class Connection extends EventEmitter {
 		);
 	}
 
-	info( message ) {
+	/**
+	 * Send customer and browser information
+	 * @param { Object } info selected form fields, customer date time, user agent and browser info
+	 */
+	sendInfo( info ) {
 		this.openSocket.then(
-			socket => socket.emit( 'message', { text: message.text, id: uuid(), meta: { forOperator: true } } ),
+			socket => socket.emit( 'message', {
+				type: 'customer-info',
+				id: uuid(),
+				meta: { ...info, forOperator: true },
```

Comment:
I agree but `forOperator` is used for other messages also and it is handled inside HUD as part of meta. Maybe we should have a different PR refactoring Calypso - HUD communication to streamline the structure and messages types.